src/background.ts

```diff
@@ -342,15 +342,40 @@ async function createWindow() {
     icon: path.join(__static, "icon.png"),
   });
 
+  let projectFilePath = "";
```

Comment:
念のためにundefinedにしておくのどうでしょう👀
```suggestion
  let projectFilePath: string | undefined = "";
```
今はローカル変数としてだけですが、いずれ外でも使われるかもみたいな。まあ気が向いたらで・・・！